shared/translations/languages/es.json

```diff
@@ -481,7 +481,7 @@
     "Don’t share to anyone. Keep the recovery words only to yourself.": "No las compartas con nadie. Guárdalas sólo para ti",
     "I have read and agree to the terms and conditions.": "He leído y acepto las condiciones.",
     "View terms and conditions.": "Mostrar las condiciones generales.",
-    "https://dfx.swiss/en/terms/": "https://dfx.swiss/es/terminos/"
+    "https://dfx.swiss/en/terms-and-conditions/": ""
```

Comment:
How is this displayed? Default value (english) or empty string?

Reply:
And if the value is just empty string?

Reply:
dang it, that shouldn't have happened though!

thx